nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,73 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the resting-state networks atlas
+    published by Allen et al. in 2011. The provided images are in MNI152 space.
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        directory where data should be downloaded and unpacked.
+    url: string, optional
+        url of file to download.
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+        - "all_unthresh_tmaps": T-maps of all 75 unthresholded components.
```

Comment:
I am not sure. The maps are very similar, but the values are different. The description page does not help: http://mialab.mrn.org/data/index.html